server/app/services/cloud/rpc_server.rb

```diff
@@ -7,13 +7,16 @@ class RpcServer
     }
 
     class Error < StandardError
-      attr_reader :code
+      attr_reader :code, :remote_backtrace
```

Comment:
Same thing re `RpcServer::Error`, the `Cloud::RpcServer::Error` doesn't have any remote backtrace.

In fact, this exception is not even used at all... all of the `Cloud::Rpc` handlers use the `RpcServer::Error`, not the `Cloud::RpcServer::Error` :confounded: